jme3-lwjgl3/src/main/java/com/jme3/system/lwjgl/LwjglWindow.java

```diff
@@ -688,4 +706,28 @@ public void destroy(boolean waitFor) {
     public long getWindowHandle() {
         return window;
     }
+
+    /**
+     * Get the window content scale, for HiDPI support.
+     *
+     * The content scale is the ratio between the current DPI and the platform's default DPI.
+     * This is especially important for text and any UI elements. If the pixel dimensions of
+     * your UI scaled by this look appropriate on your machine then it should appear at a
+     * reasonable size on other machines regardless of their DPI and scaling settings. This
+     * relies on the system DPI and scaling settings being somewhat correct.
+     *
+     * @param store A vector2f to store the result
+     * @return The window content scale
+     * @see <a href="https://www.glfw.org/docs/latest/window_guide.html#window_scale">Window content scale</a>
+     */
+    public Vector2f getWindowContentScale(Vector2f store) {
```

Comment:
Would be great if this would be actually part of some interface (JmeContext?). And also required to be implemented by LWJGL 2 (this is fairly easy there, no need to even do a dummy implementation). Of course a bit extra work already.. But would enable jME apps to scale their UIs finally. Also a callback/listener?